### ISSUE TITLE
UX: Adding a stop in a path puts it at the top

### ISSUE THREAD

**Issue by absempere** [performant-software/gbof-astro#10]
When creating a path, I expect that I'm going to start with the first stop and then add the next stop and then another -- it's a linear narrative where I start at the beginning and map my way to the end. 

The current interface does the opposite -- every new stop on the path is added to the beginning, so the path ends up in the opposite order of what I expected. I can drag and drop to reorder this, but as a default, can we add new path items to the end of a path instead of to the beginning?

**Comment by ajolipa** [comment]
Yes, this one annoys me to. I had brought it up with @jamiefolsom at some point and he said don't worry about it for now, but I agree it's not ideal. I've already put a bit of time into messing around with the Tina config and I don't think there's a straightforward way to change that setting, or at least I can't find one. So ultimately I think this is fixable but would involve creating an entirely custom component for displaying the list of places. That's definitely possible but is a bit more involved/time-consuming so I might push it down the road a bit, unless it feels like a major priority to y'all.

**Comment by absempere** [comment]
@ajolipa @jamiefolsom Let's push this down the road. We'll want this in place by the time students are using the system for class, but it isn't urgent.
